.gdbinit

```diff
@@ -0,0 +1,3 @@
+break __assert_fail
```

Comment:
I don't think we should have this file as part of the project do we?

Reply:
possible could be added to `.gitignore`